.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: Build
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      job:
+        description: 'Job to run'
+        required: true
+        default: 'build-thunderstore-package'
```

Comment:
What's this doing here?

Reply:
Whoops, leftover from copy/pasting stuff. Removed in 778991287e96b5b3861cc8a126e9c7c3623f6dc2

CI sample run: https://github.com/GeckoEidechse/Temp-NorthstarLauncher/actions/runs/4406988833
Resulting files: https://github.com/GeckoEidechse/Temp-NorthstarLauncher/releases/tag/v0.0.17